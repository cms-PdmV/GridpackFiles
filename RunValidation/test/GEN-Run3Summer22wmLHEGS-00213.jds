executable = GEN-Run3Summer22wmLHEGS-00213.sh
getenv = True
when_to_transfer_output = on_exit
+JobFlavour = "workday"
+JobBatchName = "GEN-Run3Summer22wmLHEGS-00213"
output = GEN-Run3Summer22wmLHEGS-00213.stdout
error = GEN-Run3Summer22wmLHEGS-00213.stderr
log = GEN-Run3Summer22wmLHEGS-00213.stdlog
request_memory = 8192
+SingularityImage = "/cvmfs/singularity.opensciencegrid.org/cmssw/cms:rhel8-m-m20230425"
queue

