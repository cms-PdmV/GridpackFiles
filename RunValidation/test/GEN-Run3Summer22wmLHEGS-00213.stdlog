000 (8360715.000.000) 06/03 10:09:33 Job submitted from host: <188.184.90.62:9618?addrs=188.184.90.62-9618+[2001-1458-201-e4--100-238]-9618&alias=bigbird16.cern.ch&noUDP&sock=schedd_3504972_d6e3>
...
040 (8360715.000.000) 06/03 10:11:11 Started transferring input files
	Transferring to host: <188.184.162.167:9618?addrs=188.184.162.167-9618+[2001-1458-301-4a--100-a1]-9618&alias=b7s11p8576.cern.ch&noUDP&sock=slot1_7_4812_3d3d_83404>
...
040 (8360715.000.000) 06/03 10:11:11 Finished transferring input files
...
001 (8360715.000.000) 06/03 10:11:12 Job executing on host: <188.184.162.167:9618?addrs=188.184.162.167-9618+[2001-1458-301-4a--100-a1]-9618&alias=b7s11p8576.cern.ch&noUDP&sock=startd_3282_c1ea>
...
006 (8360715.000.000) 06/03 10:11:24 Image size of job updated: 31784
	15  -  MemoryUsage of job (MB)
	15240  -  ResidentSetSize of job (KB)
...
006 (8360715.000.000) 06/03 10:14:18 Image size of job updated: 5668996
	15  -  MemoryUsage of job (MB)
	15240  -  ResidentSetSize of job (KB)
...
040 (8360715.000.000) 06/03 10:14:18 Started transferring output files
...
040 (8360715.000.000) 06/03 10:14:18 Finished transferring output files
...
005 (8360715.000.000) 06/03 10:14:18 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:02:07, Sys 0 00:00:18  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:02:07, Sys 0 00:00:18  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	43942  -  Run Bytes Sent By Job
	915  -  Run Bytes Received By Job
	43942  -  Total Bytes Sent By Job
	915  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.34        4         4 
	   Disk (KB)            :  2543           1   3076036 
	   Memory (MB)          :    15        8192      8200 

	Job terminated of its own accord at 2023-06-03T08:14:18Z.
...
