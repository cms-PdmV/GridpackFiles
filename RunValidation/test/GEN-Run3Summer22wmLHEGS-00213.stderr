WARNING: Overriding HOME environment variable with APPTAINERENV_HOME is not permitted
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
--2023-06-03 10:11:24--  https://cms-pdmv.cern.ch/mcm/public/restapi/requests/get_fragment/GEN-Run3Summer22wmLHEGS-00213/0
Resolving cms-pdmv.cern.ch (cms-pdmv.cern.ch)... 188.185.84.85
Connecting to cms-pdmv.cern.ch (cms-pdmv.cern.ch)|188.185.84.85|:443... connected.
WARNING: The certificate of 'cms-pdmv.cern.ch' is not trusted.
WARNING: The certificate of 'cms-pdmv.cern.ch' hasn't got a known issuer.
HTTP request sent, awaiting response... 200 OK
Length: 1932 (1.9K) [text/plain]
Saving to: 'GEN-Run3Summer22wmLHEGS-00213.py'

     0K .                                                     100% 38.0M=0s

2023-06-03 10:11:24 (38.0 MB/s) - 'GEN-Run3Summer22wmLHEGS-00213.py' saved [1932/1932]

WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
ERROR: Can not setup your current working area for SCRAM_ARCH: $ENV{SCRAM_ARCH}
Your current development area ${location}/${areaname}
is using a different ${areaname}/config then the one used for
/cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_4_8
./runcmsgrid.sh: line 41: cd: CMSSW_12_4_8/src: No such file or directory
03-Jun-2023 10:14:12 CEST  Initiating request to open LHE file thread0/cmsgrid_final.lhe
03-Jun-2023 10:14:12 CEST  Successfully opened LHE file thread0/cmsgrid_final.lhe
03-Jun-2023 10:14:12 CEST  Initiating request to open LHE file thread0/cmsgrid_final.lhe
03-Jun-2023 10:14:12 CEST  Successfully opened LHE file thread0/cmsgrid_final.lhe
%MSG-w LogicError:  Pythia8ConcurrentHadronizerFilter:generator@streamBeginRun  03-Jun-2023 10:14:13 CEST Run: 1 Stream: 0
::getByLabel: An attempt was made to read a Run product before endRun() was called.
The product is of type 'LHERunInfoProduct'.
The specified ModuleLabel was 'externalLHEProducer'.
The specified productInstanceName was ''.

%MSG
Begin processing the 1st record. Run 1, Event 1, LumiSection 1 on stream 0 at 03-Jun-2023 10:14:14.670 CEST
03-Jun-2023 10:14:15 CEST  Closed LHE file thread0/cmsgrid_final.lhe

------------------------------------
GenXsecAnalyzer:
------------------------------------
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
Overall cross-section summary 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Process		xsec_before [pb]		passed	nposw	nnegw	tried	nposw	nnegw 	xsec_match [pb]			accepted [%]	 event_eff [%]
0		9.730e+05 +/- 1.628e+05		0	0	0	1	1	0	0.000e+00 +/- 0.000e+00		0.0 +/- 0.0	0.0 +/- 0.0
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
Total		9.730e+05 +/- 1.628e+05		0	0	0	1	1	0	0.000e+00 +/- 0.000e+00		0.0 +/- 0.0	0.0 +/- 0.0
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Before matching: total cross section = 9.730e+05 +- 1.628e+05 pb
After matching: total cross section = 0.000e+00 +- 0.000e+00 pb
Matching efficiency = 0.0 +/- 0.0   [TO BE USED IN MCM]

After filter: final cross section = 0.000e+00 +- 0.000e+00 pb
After filter: final fraction of events with negative weights = 0.000e+00 +- 0.000e+00
After filter: final equivalent lumi for 1M events (1/fb) = 0.000e+00 +- 0.000e+00
TimeReport> Time report complete in 160.465 seconds
 Time Summary: 
 - Min event:   1.06828
 - Max event:   1.06828
 - Avg event:   1.06828
 - Total loop:  146.567
 - Total init:  13.8757
 - Total job:   160.465
 - EventSetup Lock: 0
 - EventSetup Get:  0
 Event Throughput: 0.00682283 ev/s
 CPU Summary: 
 - Total loop:     128.902
 - Total init:     8.42791
 - Total extra:    0
 - Total children: 43.9933
 - Total job:      137.352
 Processing Summary: 
 - Number of Events:  1
 - Number of Global Begin Lumi Calls:  1
 - Number of Global Begin Run Calls: 1


=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 GenXSecAnalyzer      -w GenXSecAnalyzer:                      13       13
    2 LogicError           -w Pythia8Concurren                       1        1
    3 TimeReport           -e AfterModEndJob                         1        1
    4 fileAction           -s ExternalLHEProdu                       4        4
    5 fileAction           -s ExternalLHEProdu                       1        1

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 GenXSecAnalyzer      EndJob           EndJob           EndJob
    2 LogicError           Run: 1 Stream: 0                  
    3 TimeReport           EndJob                            
    4 fileAction           Run: 1           Run: 1           Run: 1
    5 fileAction           End Run: 1                        

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                14                  14
Error                   1                   1
System                  5                   5

dropped waiting message count 0
